charts/request-ipfs/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.4.26
+appVersion: v0.4.23
```

Comment:
We squash merge our PRs such that the PR title becomes the commit message on the `main` branch. Since the PR title already includes the word "upgrade" I'm resolving this comment as No Problem :call_me_hand: